<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSettingsColors</class>
 <widget class="QWidget" name="KSettingsColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="kcfg_useCustomColors">
     <property name="title">
      <string>Custo&amp;m colors</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>40</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel1_5">
        <property name="text">
         <string>Ledger group separator</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorButton" name="kcfg_groupMarkerColor">
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Required field</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KColorButton" name="kcfg_fieldRequiredColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>221</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel1_4">
        <property name="text">
         <string>Imported transaction</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KColorButton" name="kcfg_transactionImportedColor">
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>Matched transaction</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KColorButton" name="kcfg_transactionMatchedColor">
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>152</red>
          <green>251</green>
          <blue>152</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="kcfg_transactionErroneousColor">
        <property name="color">
         <color>
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Erroneous transaction</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorButton" name="kcfg_missingConversionRateColor">
        <property name="color">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Missing conversion rate</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
